components/StripeConnectCard.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <UCard
+    :ui="{
+      divide: isStripeConnectChecked ? undefined : '',
+      header: { base: 'flex flex-wrap justify-between items-center gap-2' },
+      body: {
+        padding: isStripeConnectChecked ? undefined : '',
+      },
+    }"
+  >
+    <template #header>
+      <h3 class="font-bold font-mono">
+        Connect to a Stripe Account
+      </h3>
+      <UToggle
+        v-model="isStripeConnectChecked"
+        name="stripe"
+        label="Use a Stripe Connect account for receiving all payment"
+      />
+    </template>
+
+    <template v-if="isStripeConnectChecked">
+      <div class="flex flex-col gap-[24px]">
+        <div class="flex items-start justify-start gap-[8px] w-full">
+          <URadio
+            v-model="isUsingDefaultAccount"
+            class="w-[50%]"
+            :value="true"
+            :disabled="!!props.shouldDisableSetting"
+          >
+            <template #label>
+              <div>Use my account</div>
+              <div
+                v-if="isDefaultAccountReady"
+                class="flex flex-col items-start"
+              >
+                <div
+                  class="flex flex-col gap-[8px] mt-[12px] px-[6px] py-[4px]"
+                >
+                  <div>✅ Has Stripe Account</div>
+                  <div>{{ ` Email: ${props.stripeConnectStatusWalletMap[props.loginAddress]?.email}` }}</div>
+                </div>
+              </div>
+
+              <span v-else>
+                <UButton
+                  class="mt-2"
+                  label="Create one here"
+                  :to="{ name: 'nft-book-store-user' }"
+                  target="_blank"
+                  variant="outline"
+                />
+              </span>
+            </template>
+          </URadio>
+          <URadio
+            v-model="isUsingDefaultAccount"
+            :value="false"
+            :disabled="!!props.shouldDisableSetting"
+            class="w-[50%]"
+          >
+            <template #label>
+              <span>Use another Stripe Express account</span>
+              <div class="flex flex-col my-[10px]">
+                <UInput
+                  :value="inputWallet"
+                  :color="stripeConnectInputError ? 'rose' : 'white'"
+                  class="font-mon w-full"
+                  placeholder="like1..."
+                  @input="onStripeConnectWalletInput"
+                />
+                <span
+                  v-if="stripeConnectInputError"
+                  class="text-red-700 text-[10px]"
+                >{{ stripeConnectInputError }}</span>
+              </div>
+              <div v-if="isStripeConnectLoading" class="text-center">
+                Loading ...
+              </div>
```

Comment:
Try to make use of Nuxt UI component like `<UProgress animation="carousel" />`

Reply:
[💄Adjust style & wording](https://github.com/likecoin/nft-book-press/pull/139/commits/a6b3d1fc776e8e2e295dfe49b9e4e79b3f745315)